packages/suite-desktop-core/webpack/core.rspack.config.ts

```diff
@@ -82,24 +82,25 @@ const config: webpack.Configuration = {
         ...dependencies,
         ...devDependencies,
         'bufferutil', // optional dependency of ws lib
-        'memcpy', // optional depencency of bytebuffer lib
         'utf-8-validate', // optional dependency of ws lib
-        'osx-temperature-sensor', // optional dependency of systeminformation lib
     ],
+    externalsType: 'commonjs2',
```

Comment:
This took me a long time to figure out. Webpack detects this automatically, but Rspack couldn't automatically resolve some of these externals, and it would throw `ReferenceError: __WEBPACK_EXTERNAL_MODULE_electron__ is not defined` during runtime (in built AppImage).
Strangely, that error only manifested when chunking is on. The `require('electron')` was missing in a file where it was supposed to be (a shared chunk `share/app~bridge.js`). Without chunking options, there is only one `.js` output file for each entry point, and Rspack correctly wrote `require('electron')` there.